src/modlunit/units.cpp

```diff
@@ -552,25 +549,19 @@ void unit_init() {
     }
 #if defined(__MINGW32__)
     if (!inpfile) {
-        s = strdup(neuronhome());
-        if (s) {
-            if (strncmp(s, "/cygdrive/", 10) == 0) {
-                /* /cygdrive/x/... to c:/... */
-                Sprintf(buf, "%c:%s/lib/nrnunits.lib", s[10], s + 11);
-            } else {
-                Sprintf(buf, "%s/lib/nrnunits.lib", s);
-            }
-            inpfile = fopen(buf, "r");
-            free(s);
+        auto s = neuronhome();
```

Comment:
`s` seems to be declared before here:
https://github.com/neuronsimulator/nrn/blob/1082c4b39c330fea086ef8dc06245bd5b20c5a51/src/modlunit/units.cpp#L535
so I think that one should be changed to `std::string s;`, and the `auto` here (and below) removed.

Reply:
Those both `s` are unrelated.